lib/sensors/include/AEAT6012.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+/*
+ * Broadcom's AEAT-6012 is an absolute magnetic encoder with 12 bit resolution and SPI interface
+ * Datasheet: https://docs.broadcom.com/doc/AV02-0188EN
+ *
+ * Currently a WIP
+ *
+ * TODO:
+ * - Make driver async
+ * - Incorporate under Encoder base class
+ */
+
+namespace AEAT6012 {
```

Comment:
id rather namespace Encoder

Reply:
I'm still confused about if this encoder will eventually be grouped under the Encoder classes we have under `lib/sensors`? I thought the base Encoder class will get wrapped in the "Encoder" namespace

Reply:
so all encoder classses go under Encoder namespace. namespaces arent inherited